libs/ui/atoms/src/lib/Icon/__stories__/Icon.stories.tsx

```diff
@@ -10,8 +11,13 @@ storiesOf('Icon', module)
     <Row px={8} space={4} mt={4} flexWrap='wrap'>
       {Object.values(IconType).map((name, index) => (
         <View mt={4} key={index}>
-          <Icon name={name} />
+          <Icon size={17} name={name} />
```

Comment:
Why 17? Could it be 16?

Reply:
Yes, there can be any value which is integer.